ddtrace/internal/datadog/profiling/dd_wrapper/src/synchronized_sample_pool.cpp

```diff
@@ -0,0 +1,79 @@
+#include "synchronized_sample_pool.hpp"
```

Comment:
NIT: do we not require a header with license info in dd-trace-py ?

Reply:
No, [all these headers](https://github.com/DataDog/dd-trace-py/tree/taegyunkim/prof-10011-lockfree/ddtrace/internal/datadog/profiling/dd_wrapper/include) are missing license.